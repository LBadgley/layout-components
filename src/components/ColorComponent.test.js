import React from 'react';
import { shallow } from 'enzyme';
import Color from './ColorComponent';

describe('color component test', () => {
  it('renders the color component', () => {
    const wrapper = shallow(<Color />);
    expect(wrapper).toMatchSnapshot();
  });
});
