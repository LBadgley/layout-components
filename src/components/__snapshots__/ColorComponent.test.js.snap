// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`color component test renders the color component 1`] = `
<dl>
  <dt>
    Name
  </dt>
  <dd>
    Stil de Grain Yellow
  </dd>
  <dt>
    Hex
  </dt>
  <dd>
    #F4D35E
  </dd>
  <dt>
    RGB
  </dt>
  <dd>
    R: 244
  </dd>
  <dd>
    G: 211
  </dd>
  <dd>
    B: 94
  </dd>
</dl>
`;
